tests/boards/intel_adsp/hda/src/smoke.c

```diff
@@ -0,0 +1,189 @@
+/* Copyright (c) 2022 Intel Corporation
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <kernel.h>
+#include <ztest.h>
+#include <cavs_ipc.h>
+#include <cavs_hda.h>
+#include "tests.h"
+
+#define IPC_TIMEOUT K_MSEC(100)
+#define STREAM_ID 3U
+#define FIFO_SIZE 256
+#define TRANSFER_SIZE 256
+#define TRANSFER_COUNT 8
+
+__attribute__((section(".dma_buffers"), aligned(128))) uint8_t hda_fifo[FIFO_SIZE];
```

Comment:
As mentioned earlier, everything will work here if you drop the section and just make an aligned static buffer.

Reply:
I'll try it, I had many issues, some of this stuff is left over because at the end of debugging I had a working version and wanted to move on to creating a dma driver on top of this. I tried reverting the linker script changes made above and the test failed so once it worked... I left well enough alone